app/facades/quote_facade.rb

```diff
@@ -0,0 +1,10 @@
+class QuoteFacade
+  def self.quote_search(query)
```

Comment:
Okay so since we want to access both `:totalCount` and the quote attributes that are nested within `:results` we can pass two things. 
```
def self.quote_search(query)
    search = QuotesService.search_by_keyword(query)
    quotes = search[:results].map do |quote_data|
      Quote.new(quote_data)
    end
    [search[:totalCount], quotes]
  end
```
Now we will have access to a total_count and quote attributes